events/syscall/disallowed_ssh_connection.go

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+	"os/exec"
+	"github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(
+	DisallowedSSHConnection,
+	events.WithDisabled(), // this rule is not included in falco_rules.yaml (stable rules), so disable the action
+)
+
+func DisallowedSSHConnection(h events.Helper) error {
+	cmd := exec.Command("timeout", "1s", "ssh", "user@example.com", "-p", "22")
```

Comment:
Check if ssh is present?

Reply:
Yes @FedeDP, Let me modify it

Reply:
@FedeDP I made changes and rule triggering successfully and throwing error too when ssh is not present.

Reply:
@leogr as per our discussion [here](https://github.com/falcosecurity/event-generator/pull/144#discussion_r1562005666), This rule is also not triggering on localhost 
![Screenshot from 2024-04-12 10-01-35](https://github.com/falcosecurity/event-generator/assets/117383853/ff4d5157-7169-4d3d-9740-241bba2b423e)

Reply:
Let's use `example.com`, if there's no alternative. 
@FedeDP wdyt?

Reply:
@FedeDP If using example.com looks good to you, then we are good to go with this PR, please take a look.